packages/kernel-6.12/kernel-6.12.spec

```diff
@@ -435,20 +408,11 @@ install -p -m 0644 %{S:301} %{buildroot}%{_cross_bootconfigdir}/05-vmware.conf
 %{_cross_includedir}/xen/*
 
 %files devel
-%dir %{_cross_ksrcdir}
+%{_cross_usrsrc}/kernels/%{kmajor}
+%{_cross_ksrcdir}
```

Comment:
should this still be `%dir` or do we want to package in everything under `%{_cross_ksrcdir}` for devel?

Reply:
We want to include everything underneath it.